pyproject.toml

```
@@ -91,7 +91,7 @@ Documentation = "https://napari.org"
 
 [project.optional-dependencies]
 pyside2 = [
-    "PySide2>=5.13.2,!=5.15.0 ; python_version < '3.12'",
+    "PySide2>=5.13.2,!=5.15.0 ; python_version < '3.12' and platform_machine == 'x86_64'",
```

```suggestion
    "PySide2>=5.13.2,!=5.15.0 ; python_version < '3.11' and platform_machine == 'x86_64'",
```
None of the wheels on pypi support 3.11